Makefile

```diff
@@ -16,56 +15,57 @@ PYTEST := $(PYTHON) -m pytest
 PYRIGHT := $(PYTHON) -m pyright
 RUFF := $(PYTHON) -m ruff
 
+COG_PYTHON_SOURCE := $(shell find python/cog -type f -name '*.py')
+COG_VERSION := $(shell $(PYTHON) -m setuptools_scm 2>/dev/null)
+
+ifndef COG_VERSION
+$(error Could not determine a version for cog! Did you `pip install -e '.[dev]'` first?)
+endif
```

Comment:
does this break running `make cog` locally?

Reply:
this seems to work fine on my machine, so I wonder if the hint is accurate

Reply:
This shouldn't have ever broken `make cog` locally, because you're expected to have done `pip install -e '.[dev]'` before you do that.

Reply:
this hasn't been necessary though...? I've literally never done that in the past year and everything has worked fine...